laws/src/main/scala/cats/laws/InjectKLaws.scala

```diff
@@ -0,0 +1,12 @@
+package cats
+package laws
+
+trait InjectKLaws[F[_], G[_]] {
+  def injectK: InjectK[F, G]
+
+  def injectKRoundTripInj[A](fa: F[A]): IsEq[Option[F[A]]] =
+    (injectK.prj compose injectK.inj).apply(fa) <-> Some(fa)
+
+  def injectKRoundTripPrj[A](ga: G[A]): IsEq[Option[G[A]]] =
+    injectK.prj(ga).map(fa => injectK.inj(fa)) <-> Some(ga)
```

Comment:
Yes! This commit is goofy-- I cherry picked it from ages ago and I'm reworking it now.

That being said, how do you check this kind of behavior with discipline? Would it suffice to just have the `injectKRoundTripInj` and nix `injectKRoundTripPrj`?